### ISSUE TITLE
Tabs not functioning

### ISSUE THREAD

**Issue by mrsideshowjack** [carbon-design-system/carbon-components-vue#385]
## When importing the tabs files directly

> The tab group does render and content inside each tab does switch

So getting these two errors with `cv-tabs` and can’t get anything from `@tab-selected`,  `"@carbon/vue": "^2.0.0-beta.2"`

```
[Vue warn]: Error in v-on handler: "TypeError: handler.apply is not a function"

found in

---> <CvTabs> at node_modules/@carbon/vue/src/components/cv-tabs/cv-tabs.vue
         <Main> at src/App.vue
           <Root>
```
```
[Vue warn]: Multiple root nodes returned from render function. Render function should return a single root node.

found in

---> <CvDropdownInner> at node_modules/@carbon/vue/src/components/cv-dropdown/_cv-dropdown-inner-c10.vue
       <CvDropdown> at node_modules/@carbon/vue/src/components/cv-dropdown/cv-dropdown.vue
         <CvTabs> at node_modules/@carbon/vue/src/components/cv-tabs/cv-tabs.vue
             <Main> at src/App.vue
               <Root>
```

## When importing all components from main.js

> tabs to not render, all consent inside each tabs is displayed

Additionally getting the error `Unknown custom element: <cv-tab> - did you register the component correctly?` on codesandbox, while `cv-button` imports and functions correctly.

## Steps to reproduce the issue
Use tabs
[Codesandbox link 1](https://codesandbox.io/s/91ooz1n48r)
[Codesandbox link 2](https://codesandbox.io/s/91ooz1n48r)

**Comment by lee-chase** [comment]
@mrsideshowjack  Fixed in 2.0.3

**Comment by lee-chase** [comment]
There is no prop on cv-tabs but an event tabs-selected. 

On individual cv-tab elements, you can set the 'selected' attribute

export default {
 data() {
   return {
     dataTabSelected: 1
   };
 },
 methods: {
   actionTabSelected(val) {
     this.dataTabSelected = val;
   }
 }
};
